condor/runHHH6bPNetAK4.py

```diff
@@ -52,7 +52,7 @@
     '1': '1',
     '2': '1',
     '3': '1',
-    '4': 'nJet > 2',
+    '4': 'nJet >= 4',
```

Comment:
---> Why did we change preselection wrt v33? That is too many changes in one goal no?
Are you changing on account of 4b2t? But will change our control region...

Reply:
I fixed it because it's basically picked up in the NanoNN framework:

https://github.com/mstamenk/NanoNN/blob/mstamenk-update-v34/python/producers/hhh6bProducerPNetAK4.py#L2099

So it's just to harmonise and reduce the number of events the framework is ran on

Reply:
ok! For the record (in the case we need to investigate) was this by chance in v33 by being in your local area?

Reply:
@mstamenk answer for future reference, I am merging